Sources/Brave/Frontend/Shields/ShieldsViewController.swift

```diff
@@ -213,7 +213,7 @@ class ShieldsViewController: UIViewController, PopoverContentComponent {
   /// Groups the shield types with their control and global preference
   private lazy var shieldControlMapping: [(BraveShield, AdvancedShieldsView.ToggleView, Preferences.Option<Bool>?)] = [
     (.AdblockAndTp, shieldsView.advancedShieldView.adsTrackersControl, Preferences.Shields.blockAdsAndTracking),
-    (.SafeBrowsing, shieldsView.advancedShieldView.blockMalwareControl, Preferences.Shields.blockPhishingAndMalware),
+    (.SafeBrowsing, shieldsView.advancedShieldView.blockMalwareControl, Preferences.Shields.googleSafeBrowsing),
```

Comment:
We should remove the `SafeBrowsing` shield entirely as it'll no longer be domain-specific